io.jamulus.Jamulus.desktop

```diff
@@ -0,0 +1,24 @@
+[Desktop Entry]
```

Comment:
What is the change here as compared to upstream's file? If it modifies some keys, `desktop-file-edit` can be used in post-install instead.

Reply:
The idea here was to basically add a second launch option. This would eventually go upstream, but I think it could start like this and be migrated later, same as with the metafile, and as mentioned in https://github.com/flathub/flathub/pull/2101#issuecomment-775027117

Reply:
:+1: